visual_behavior/ophys/dataset/visual_behavior_ophys_dataset.py

```diff
@@ -107,17 +109,23 @@ def get_timestamps(self):
 
     timestamps = LazyLoadable('_timestamps', get_timestamps)
 
-    def get_timestamps_stimulus(self):
-        self._timestamps_stimulus = self.timestamps['stimulus_frames']['timestamps']
-        return self._timestamps_stimulus
+    def get_stimulus_timestamps(self):
```

Comment:
@dougollerenshaw these changes to the naming of the timestamps variables could mess with things, if anyone is using this class for their analysis. I just sent a message to the analysis channel on Slack to see if anyone depends on this, and generally warn everyone about this big merge

Reply:
make sense. The new convention is definitely more readable, though.